lib/assets/javascripts/turbograft/turbolinks.coffee

```diff
@@ -117,17 +333,22 @@ class window.Turbolinks
   @loadPage: (url, xhr, options = {}) ->
     triggerEvent 'page:receive'
     options.updatePushState ?= true
-
-    if doc = processResponse(xhr, options.partialReplace)
+    if upstreamDocument = processResponse(xhr, options.partialReplace)
       reflectNewUrl url if options.updatePushState
-      nodes = changePage(extractTitleAndBody(doc)..., options)
+      nodes = changePage(
```

Comment:
This is still going to update the doc + run inline scripts before new JS is loaded.